deployment/aks-periscope.yaml

```diff
@@ -132,6 +134,14 @@ metadata:
 data:
   DIAGNOSTIC_NODELOGS_LIST: /var/log/azure/cluster-provision.log /var/log/cloud-init.log
 ---
+apiVersion: v1 
+kind: ConfigMap 
+metadata:
+  name: clustertype-config
+  namespace: aks-periscope
+data:
+  CLUSTER_TYPE: managedCluster
```

Comment:
💡 This default value of `managedClsuter` is unused i.e.  never existed, so lets keep `defaultValue` keep this as **empty** **because** within your code you only check for the `connectedCluster` there is not comparison of `managedCluster` anywhere in the code, so this used `defaultValue` kind of looses its use. 

Could do like what `secret` flags do:

` CLUSTER_TYPE: # <used as flag arc cluster>`

Reply:
@Tatsinnit , just pushed new changes that fixes this. If it looks better now, we can resolve this thread.

Reply:
Thanks @sophsoph321 thanks, I have a minor recommendation which you could commit as one of last commits to use much better scaffolding name I was thinking:

* `# <custom flag>` - I am thinking of this as a scaffolding for the other consuming tools which use a `replace` and use it for consumptions purposes.

Reply:
Sure. I just pushed a change that accounts for this change to use `# <custom flag>` as the clusterType value